### PR TITLE
RegularPolygonMarker docstring

### PR DESCRIPTION
**Pull request by BibMartin**
addresses #281

### CODE REVIEW

**Comment by BibMartin** [comment]
adresses #281

File: folium/features.py [hunk 1]
```diff
@@ -49,12 +49,40 @@ def __init__(self, url, name=None,
...
+        radius: int, default 15
+            Marker radius, in pixels
+        popup: string or folium.Popup, default None
+            Input text or visualization for object. Can pass either text,
+            or a folium.Popup object.
+            If None, no popup will be displayed.
+
+        Returns
+        -------
+        Polygon marker names and HTML in obj.template_vars
```
Comment (ocefpaf) [inline folium/features.py]: Can you add a link to Leaflet Data Vis Framework with the phrase:

`For more information see URL`

File: folium/features.py [hunk 2]
```diff
@@ -49,12 +49,40 @@ def __init__(self, url, name=None,
...
+            Marker fill color
+        fill_opacity: float, default 1
+            Marker fill opacity
+        number_of_sides: int, default 4
+            Number of polygon sides
+        rotation: int, default 0
+            Rotation angle in degrees
+        radius: int, default 15
+            Marker radius, in pixels
+        popup: string or folium.Popup, default None
```
Comment (ocefpaf) [inline folium/features.py]: `string` it not working yet, right?

**Comment by ocefpaf** [comment]
:+1:

File: folium/features.py (hunk 2 above)
Comment (BibMartin) [inline folium/features.py]: Done and squashed

File: folium/features.py [hunk 3]
```diff
@@ -49,12 +49,40 @@ def __init__(self, url, name=None,
...
-    def __init__(self, location, popup=None,
-                 color='black', opacity=1, weight=2,
+    def __init__(self, location, color='black', opacity=1, weight=2,
                  fill_color='blue', fill_opacity=1,
-                 number_of_sides=4, rotation=0, radius=15):
-        """
-        TODO docstring here
+                 number_of_sides=4, rotation=0, radius=15, popup=None):
+        """Custom markers using the Leaflet Data Vis Framework.
+        For more information, see https://humangeo.github.io/leaflet-dvf/
```
Comment (ocefpaf) [inline folium/features.py]: I was thinking that this should be at the bottom.  That is consistent with the other docstrings I am writing and makes sense.  You read what is here first, then you follow the URL for more info :wink:

**Comment by BibMartin** [comment]
> I was thinking that this should be at the bottom. That is consistent with the other docstrings I am writing and makes sense. You read what is here first, then you follow the URL for more info :wink:

Done and squashed.

> string it not working yet, right?

That's true. BUt it's coming from #265 in a few days (I promise).

**Comment by ocefpaf** [comment]
> That's true. But it's coming from #265 in a few days (I promise).

No problem.  I'd rather have that added in all docstrings now than implementing it later and forgetting about the doscstrings.

**Pull request merged**